docker-compose.yml

```diff
@@ -300,3 +295,17 @@ services:
     volumes:
       - ./resources/kibana/kibana.yml:/usr/share/kibana/config/kibana.yml
 
+  ogc-api-records:
+    image: geonetwork/gn-cloud-ogc-api-records-service:4.2.2
```

Comment:
ideally: should be "latest" on master, but wondering also if this should be a `geonetwork/gn-cloud-ogc-api-records-service` or a `georchestra/gn-cloud-ogc-api-records-service`, see https://github.com/georchestra/georchestra/issues/3931

Reply:
I can set latest, 
otherwise not idea of which image to use (geonetwork or georchestra)

Reply:
latest does not exist ..

Reply:
I think we tested it to work with this tag; seeing how much trouble we have making this all work together, staying on a floating tag sounds unfeasible to me

Reply:
not possible i guess : 
georchestra/gn-cloud-ogc-api-records-service:latest ---> really old (1 year ago)
geonetwork/gn-cloud-ogc-api-records-service:latest ---> do not exist

Reply:
(I meant 4.2.2)